pints/interfaces/__init__.py

```diff
@@ -0,0 +1,12 @@
+#
+# Root of the interfaces module.
+# Provides a number of toy models and logpdfs for tests of Pints' functions.
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+
+from ._stan import StanLogPDF                                       # noqa
```

Comment:
This would render the entire `interfaces` module useless if the `stan` interface doesn't work.

Some alternative options:

- Long form: `pints.interfaces.stan.StanThing`, then later `pints.interfaces.pymc3.OtherThing` etc.
- Short form, with some hint in name: `pints.interface_stan.StanThing` or `pints.stan_interface.StanThing`
- Simple form: `pints.stan.StanThing`

Reply:
If we go the `pints.interface.stan.StanThing` route the `pints.interfaces` module would be pretty much just a placeholder, so it would not import the stan module in its init.
Users would do e.g.
```
import pints.interfaces.stan as stan

print(stan.StanThing)
```

Reply:
good idea. Done